cfpb/cfgov-refresh
# Update Capital Framework and use family-linked webfonts

This PR will eliminate the use of the old `u-webfont-` mixins, replacing them with the standard `font-style` and `font-weight` CSS properties that are now able to be used with the included upgrade to the latest version of cf-typography.

## Removals

- Outdated styles
- PDFreactor fonts that are no longer needed

## Changes

- Converted `u-webfont-` mixin calls to `font-style` or `font-weight` properties

## Testing

1. Pull branch
1. `npm install`
1. `gulp`
1. Browse through your local site and observe no changes from the production site

## Todos

- [x] Before this gets merged, we need another release of CF and to increment the versions updated here.

## Checklist
